module-2/backend/week_two.md

```diff
@@ -4,17 +4,25 @@ Fork this respository. Answer the questions to the best of your ability. Try to
 
 Note: When you're done, submit a PR. 
 
-1. At a high level, what is ActiveRecord? What does it do/allow you to do?
+1. At a high level, what is ActiveRecord? What does it do/allow you to do? 
+      ActiveRecord is a frame work that allows you to make databases, motify them, and correlate these databases to visual    
+      representations of your program
 2. Assume you have the following model:
 
 ```ruby
 class Team << ActiveRecord::Base
 end
 ```
+  
+What are some methods you can call on `Team`? If these methods aren't defined in the class, how do you have access to them? 
+      #min 
+      #max 
+      #first 
+      #last 
+      just to name a few. This model is inheriting from Active Record so you have active record methods available. 
 
-What are some methods you can call on `Team`? If these methods aren't defined in the class, how do you have access to them?
-
-3. Assume that in your database, a team has the following attributes: "id", "name", owner_id". How would you find the name of a team with an id of 4? Assuming your class only included the code from question 2, how could you find the owner of the same team?
+3. Assume that in your database, a team has the following attributes: "id", "name", owner_id". How would you find the name of a team with an id of 4? Assuming your class only included the code from question 2, how could you find the owner of the same team?  
+      Team.find(4).name
```

Comment:
Yup! Regarding the second question, you'd need something like this:

team_4_owner_id = team.find(4).owner_id
owner = Owner.find(team_4_owner_id)